packages/next/src/server/config.ts

```diff
@@ -1071,6 +1071,14 @@ export default async function loadConfig(
 
   const path = await findUp(CONFIG_FILES, { cwd: dir })
 
+  if (process.env.__NEXT_TEST_MODE) {
+    if (path) {
+      Log.info(`Loading config from ${path}`)
+    } else {
+      Log.info('No config file found')
+    }
+  }
```

Comment:
Might be good to also log the text fixture directory so it can be compared with the config path

Reply:
That has been logged by https://github.com/vercel/next.js/blob/jude/which-config-are-we-using/test/lib/next-test-utils.ts#L246